server/project/parse.ts

```diff
@@ -77,3 +78,40 @@ export function parseRouterFunctions(code: string) {
 
   return routerFunctions;
 }
+
+export function createOperation(code: string) {
+  const routeRegExp =
+    /@Router\.(get|post)\("([^"]+)"\)\s+async\s+(\w+)\(((?:\s*\w+\s*:\s*\w+\s*,?\s*)+)\)\s*{/;
+  const match = routeRegExp.exec(code);
+
+  if (match) {
+    const method = match[1];
+    const endpoint = match[2];
+    const functionName = match[3];
+    const params = match[4]
+      .split(",")
+      .map((param) => {
+        const paramName = param.trim().split(":")[0];
+        return paramName;
+      })
+      .filter((param) => param !== "" && !param.startsWith("session"));
+
+    let fieldsString = "{";
+    params.forEach((param, index) => {
+      fieldsString += `${param}: "input"`;
+      if (index !== params.length - 1) {
+        fieldsString += ", ";
+      }
+    });
+    fieldsString += "}";
+
+    const separatedFunctionName = functionName
+      .replace(/([A-Z])/g, " $1")
+      .trim();
+    const capitalizedFunctionName =
+      separatedFunctionName.charAt(0).toUpperCase() +
+      separatedFunctionName.slice(1);
+
+    return `\t{\n\t\tname: "${capitalizedFunctionName}",\n\t\tendpoint: "/api${endpoint}",\n\t\tmethod: "${method.toUpperCase()}",\n\t\tfields: ${fieldsString},\n\t},`;
```

Comment:
Try leveraging `JSON.stringify` here perhaps for a slightly better looking result. It should support formatting as well.